File: SobaBank/src/app/modeles/demandeOuvertureCompte.ts
import {Demande} from './demandes';
import {Agent} from './agent';

import {Client} from './client'

import { User } from './user';

export class DemandeOuvertureCompte{
	constructor(public id: number,
		public dateDemande: Date,
		public isAffected: boolean,
		public status: string,
		public isValid: boolean,
		public dateAffectation: Date,
		public client: Client,
		public idAgent: number,
		
		
		){
	}
}

export const demandesOuverture: DemandeOuvertureCompte[] = [

	new DemandeOuvertureCompte(1,new Date(),
		false,
		null,
		false,
		null,
		null,
		null
		),

	new DemandeOuvertureCompte(2,new Date(),
		false,
		null,
		false,
		null,
		null,
		null
		),

	new DemandeOuvertureCompte(3,new Date(),
		false,
		null,
		false,
		null,
		null,
		null
		),

	new DemandeOuvertureCompte(4,new Date(),
		false,
		null,
		false,
		null,
		null,
		null
		)
]

File: sobabank_back/src/main/java/com/bl/model/DemandeOuvertureCompte.java
package com.bl.model;

import java.util.Date;


public class DemandeOuvertureCompte {
	private Long id;
	private Date dateDemande;
	private Boolean isAffected;
	private String Status;
	private Boolean isValid;
	private Date dateAffectation;
	private Client client;
	private Agent agent;
	
	public DemandeOuvertureCompte() {
	}

	public Long getId() {
		return id;
	}

	public void setId(Long id) {
		this.id = id;
	}

	public Date getDateDemande() {
		return dateDemande;
	}

	public void setDateDemande(Date dateDemande) {
		this.dateDemande = dateDemande;
	}

	public Boolean getIsAffected() {
		return isAffected;
	}

	public void setIsAffected(Boolean isAffected) {
		this.isAffected = isAffected;
	}

	public String getStatus() {
		return Status;
	}

	public void setStatus(String status) {
		Status = status;
	}

	public Boolean getIsValid() {
		return isValid;
	}

	public void setIsValid(Boolean isValid) {
		this.isValid = isValid;
	}

	public Date getDateAffectation() {
		return dateAffectation;
	}

	public void setDateAffectation(Date dateAffectation) {
		this.dateAffectation = dateAffectation;
	}

	public Client getClient() {
		return client;
	}

	public void setClient(Client client) {
		this.client = client;
	}

	public Agent getAgent() {
		return agent;
	}

	public void setAgent(Agent agent) {
		this.agent = agent;
	}
	
	



}
